### PR TITLE
Update towards 1.9.2

### PR DESCRIPTION
**Pull request by oleg-nenashev**
- Changelog
- Parent POM and dependency update

@reviewbybees @EltonAlvess

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ Below you can view changelogs for the trunk version of the Windows Agent Install
 This file also provides links to Jenkins versions, which bundle the released versions.
 See [Jenkins changelog](https://jenkins.io/changelog/) for more details.
 
+## 1.9.2
+
+Release date: (Nov 03, 2017)
+
+* [JENKINS-47015](https://issues.jenkins-ci.org/browse/JENKINS-47015) -
+Performance: Do not try to update `jenkins-slave.exe` on Unix agents when they connect.
```
Comment (dwnusbaum) [inline CHANGELOG.md]: Is it really a performance improvement, or just a bugfix?

**Comment by oleg-nenashev** [comment]
@dwnusbaum it is a performance issue, because it contributes to [JENKINS-27514](https://issues.jenkins-ci.org/browse/JENKINS-27514). On massive agent reconnects this thing causes an extra task for each agent to the thread pool, which has no growth limit. So in the worst case you may easily get e.g. a hundred of extra threads spawned.

**Comment by oleg-nenashev** [comment]
@reviewbybees done
@dwnusbaum can remove performance if you prefer

**Comment by dwnusbaum** [comment]
@oleg-nenashev Ah ok, I didn't click through your comment on JENKINS-47015 that mentioned JENKINS-27514. The changelog looks good as is, thanks for explaining!

**Pull request merged**